frontend/apps/tamagotchi-battle/src/features/battle/hooks/use-battle.ts

```diff
@@ -87,23 +87,24 @@ export function useInitBattleData() {
         } = data;
 
         if (details.isSome && !details.unwrap().to.eq(0)) {
-          // console.log(payload.toHuman());
-          // alert.error(`${payload.toHuman()}`, { title: 'Error during program execution' });
-        } else {
-          if (metadata.types.handle.output) {
-            const decodedPayload = metadata.createType(metadata.types.handle.output, payload).toJSON();
-            if (
-              decodedPayload &&
-              typeof decodedPayload === 'object' &&
-              Object.keys(decodedPayload).includes('roundResult')
-            ) {
-              const notification = Object.values(decodedPayload)[0] as RoundDamageType;
-
-              if (currentPairIdx === +notification[0]) {
-                // console.log({ decodedPayload });
-                setRoundDamage(notification);
-              }
-            }
+          return;
+        }
+
+        if (!metadata.types.handle.output) {
```

Comment:
Better to explicitly check for `null`, since `metadata.types.handle.output` is a number

Reply:
ok makes sense